navit/android/src/org/navitproject/navit/NavitGraphics.java

```diff
@@ -646,7 +646,7 @@ private void setmActivity(final Navit navit) {
 
     private native void motionCallback(long id, int x, int y);
 
-    private native String getCoordForPoint(int x, int y, boolean absoluteCoord);
+    private static native String getCoordForPoint(int x, int y, boolean absoluteCoord);
```

Comment:
Hi, can you give a motivation for this, does it solve or change anything ? thx.

Reply:
Hi, I searched a bit further and you have a point in changing it to static when it comes to MIM as described here  https://github.com/navit-gps/navit/pull/964. Just if you change it to static you will have to change jobject to jclass in the signature there https://github.com/navit-gps/navit/blob/trunk/navit/android.c#L325 for consistency, thx.

Reply:
Hi, yes, that was the reason for this change. This method does not access any member of the class.
I agree it has nothing to do with the feature in this PR. I just fixed this on the way while browsing through [the code I wrote earlier](https://github.com/navit-gps/navit/pull/794/files#diff-de01e9991ecc98ddc67d26f95a4e22f3) in the previously related PR #794.

But you're right, this is not required to implement the feature described in this current PR.

Reply:
Thanks for the review, I've just changed this.